### PR TITLE
15552 graphql docs

### PR DESCRIPTION
**Pull request by arthanson**
### Fixes: #15552 

Just updates the graphql-api docs for the new filtering syntax

### CODE REVIEW

File: docs/integrations/graphql-api.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ For more detail on constructing GraphQL queries, see the [Graphene documentation
 The GraphQL API employs the same filtering logic as the UI and REST API. Filters can be specified as key-value pairs within parentheses immediately following the query name. For example, the following will return only sites within the North Carolina region with a status of active:
 
 ```
-{"query": "query {site_list(region:\"north-carolina\", status:\"active\") {name}}"}
+{"query": "query {site_list(filters: {region: "us-nc\", status: "active"}) {name}}"}
```
Comment (jeremystretch) [inline docs/integrations/graphql-api.md:57]: Remove errant backslash, drop encapsulating dictionary, and clean up formatting.
```suggestion
query {
  site_list(filters: {region: "us-nc", status: "active"}) {
    name
  }
}
```

File: docs/integrations/graphql-api.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ In addition, filtering can be done on list of related objects as shown in the fo
   device_list {
     id
     name
-    interfaces(enabled: true) {
+    interfaces(filters: {enabled: true}) {
```
Comment (jeremystretch) [inline docs/integrations/graphql-api.md:66]: This doesn't appear to work (with `strawberry-graphql-django` v0.34.0). I get the following error:

> Unknown argument 'filters' on field 'DeviceType.interfaces'.

**Review by jeremystretch — CHANGES_REQUESTED** [review]

File: docs/integrations/graphql-api.md (hunk 2 above)
Comment (jeremystretch) [reply docs/integrations/graphql-api.md:66]: Oh, does this depend on #15557?

Comment (arthanson) [reply docs/integrations/graphql-api.md:66]: Yes - it depends on that being merged.

**Review by jeremystretch — APPROVED** [review]

**Pull request merged**